src/const.py

```diff
@@ -25,6 +25,7 @@
 RGW_CONF_TMPL = f'{RGW_INSTALL_PATH}/conf/cortx_{COMPONENT_NAME}.conf'
 # e.g RGW_CONF_TMPL will be /opt/seagate/cortx/rgw/cof/cortx_rgw.conf
 RGW_CONF_FILE = f'cortx_{COMPONENT_NAME}.conf'
+BACKEND_STORE = 'motr'
```

Comment:
cant hardcode btw, use it from cortx_rgw.conf

Reply:
Got it, based on this added comment https://github.com/Seagate/cortx-rgw-integration/pull/32/files#r810778639,
after addressing that am good